attributes/default.rb

```diff
@@ -16,13 +16,14 @@
 #
 
 default['omnibus'].tap do |omnibus|
-  omnibus['build_user']         = 'omnibus'
-  omnibus['build_user_home']    = nil
-  omnibus['install_dir']        = nil
-  omnibus['toolchain_name']     = 'omnibus-toolchain'
-  omnibus['toolchain_version']  = '0.0.1'
-  omnibus['git_version']        = '2.6.2'
-  omnibus['ruby_version']       = '2.1.5'
+  omnibus['build_user']               = 'omnibus'
+  omnibus['build_user_home']          = nil
+  omnibus['install_dir']              = nil
+  omnibus['use_omnibus_toolchain']    = true
```

Comment:
Thoughts on skipping this tunable? I sort of feel we this should not be choice...a platform either has toolchain support or it doesn't!